src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/migration/MigrationAccountKey.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 The caver-java-ext-kas Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package xyz.groundx.caver_ext_kas.kas.wallet.migration;
+
+/**
+ * MigrationAccountKey represents a type of private key.
+ * @param <T> Basic data types that make up the various types of Migration Account Key types, e.g. String, String[], List<String[]>
+ */
+public abstract class MigrationAccountKey<T> {
```

Comment:
Why did you use a generic type?

Reply:
I used generic to deal various types of key (String, String[], List<String[]). :) 
Any better methodology is always welcome. Let me know if you have any better ideas!